containers/event-db-migrations/entry.sh

```diff
@@ -40,7 +40,21 @@ fi
 # Add historic data from previous funds
 directory="./historic_data"
 
+if [ -d "$directory" ];
+then
 for file in $(ls "$directory"/*.sql | sort); do
   echo "Adding historic data from $file"
   psql -U $PGUSER -d $PGDATABASE -h $PGHOST -p $PGPORT -f "$file"
 done
+fi
+
+# Add test data
+directory="./test_data"
+
+if [ -d "$directory" ];
+then
+for file in $(cd "$directory" && ls *.sql | sort -n); do
```

Comment:
It's best practice to use `find` in these scenarios. See an example here: https://github.com/input-output-hk/catalyst-core/pull/349/files#diff-fe4e0fde017cc32263b0093d0e53b072b171d4567f61332d7dcc89230c1d2d57R142

Reply:
fixed